### ISSUE TITLE
As a user I want social alerts for things  like a friend badging a story

### ISSUE THREAD

**Issue by wilbertom** [denverfoundation/storybase#971]

**Comment by mgrayinnovation** [comment]
What's possible in the accounts section of a user?  Are messages sent here in anyway? Or only to the users email?  Could an option be created in Accounts so that a user could chose to click a box to indicate that they would like emails sent when a story is badged, just as they click a box to determine if they want system emails sent ... they could also select the specific badges that they want to hear about.

**Comment by wilbertom** [comment]
@mgrayinnovation I think this might be too big to take on during the first iteration. I think we can plan for it after this iteration. The alert email isn't bad, the problem is that our users don't have "friends".

**Comment by mgrayinnovation** [comment]
Agreed.  Moving this to P7 ... as we get more users on the system, sharing stories etc.  We'll want to explore this.  Right now, our best bet is to focus on how readers can filter the badges, which is part of what we're working on in P1.  Thanks for pointing this out.
